test/Microsoft.AspNetCore.Testing.Tests/Microsoft.AspNetCore.Testing.Tests.csproj

```diff
@@ -7,6 +7,8 @@
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
     <!-- allow skipped tests -->
     <NoWarn>$(NoWarn);xUnit1004</NoWarn>
+    <!-- allow unused theory parameters -->
+    <NoWarn>$(NoWarn);xUnit1026</NoWarn>
```

Comment:
We could, but this was easier. Most of the tests that trigger this aren't actually using the theory parameters on purpose. e.g. https://github.com/aspnet/Testing/blob/dev/test/Microsoft.AspNetCore.Testing.Tests/ConditionalTheoryTest.cs. We are actually testing ConditionalTheoryAttribute itself.